agent/installer/bundle_downloader_test.go

```diff
@@ -84,6 +84,25 @@ var _ = Describe("Byohost Installer Tests", func() {
 				mi.Get)
 			Expect(err).ShouldNot(HaveOccurred())
 		})
+		It("Should create and rename dir correctly after successful download", func() {
+			bd.repoAddr = "repo.ccoomm/r/"
+			bd.downloadPath = filepath.Join(bd.downloadPath)
+			err := bd.DownloadFromRepo(
+				normalizedOsVersion,
+				k8sVersion,
+				testTag,
+				mi.Get)
+			Expect(err).ShouldNot((HaveOccurred()))
+			_, err = os.Stat(bd.GetBundleDirPath(k8sVersion, testTag))
```

Comment:
**Nit** - We should hard code the expected values in the test (instead of calling `GetBundleDirPath`. This will
- Improve readability. Because now readers will know what to expect
- Improve test resilience. If there is a bug in `GetBundleDirPath`, it won't affect a huge number of tests.